data/assets/scene/digitaluniverse/grids.asset

```diff
@@ -70,19 +78,23 @@ local ecliptic = {
     Transform = {
         Scale = {
             Type = "StaticScale",
-            Scale = 9.46377307652E17;
+            Scale = 9.46377307652E17
+        }, 
+        Rotation = {
```

Comment:
Isn't this the exact same rotation as above? What is it? Maybe it should be parented to the same thing, or at least save this as a local constant in the file for readability.